templates/types/streaming/nextjs/app/components/ui/chat/chat-message/codeblock.tsx

```diff
@@ -111,26 +114,18 @@ const CodeBlock: FC<Props> = memo(({ language, value }) => {
           </Button>
         </div>
       </div>
-      <SyntaxHighlighter
-        language={language}
-        style={coldarkDark}
-        PreTag="div"
-        showLineNumbers
-        customStyle={{
-          width: "100%",
-          background: "transparent",
-          padding: "1.5rem 1rem",
-          borderRadius: "0.5rem",
-        }}
-        codeTagProps={{
-          style: {
-            fontSize: "0.9rem",
-            fontFamily: "var(--font-mono)",
-          },
-        }}
-      >
-        {value}
-      </SyntaxHighlighter>
+      <pre className="border border-zinc-700">
+        <code
+          ref={codeRef}
+          className={`language-${language}`}
+          style={{
+            fontFamily:
+              'Consolas, Monaco, "Andale Mono", "Ubuntu Mono", monospace',
```

Comment:
@can't we just use the variable?
```suggestion
            fontFamily: "var(--font-mono)",
```